service-python/assessclaimdc7101/src/lib/bp_calculator.py

```diff
@@ -73,6 +98,6 @@ def bp_reader(request_body):
               "twoYearsBpCount": len(bp_readings_in_past_two_years),
               "oneYearBpCount": len(bp_reading_in_past_year),
               "twoYearsElevatedBpCount": len(elevated_bp_in_past_two_years),
-              "totalBpCount": len(request_body["evidence"]["bp_readings"])}
+              "totalBpCount": len(deduplicated_bp_readings)}
 
     return result
```

Comment:
I would add some info on what is deduplicated in evidence summary so we can see it in logs and database.

Reply:
I added "lighthouseDuplicateBpCount" to evidence summary